### PR TITLE
Revamp public readme and contribution guide

### PR DESCRIPTION
**Pull request by CharlVS**
Revamp project README to add a description, updated API info, screenshots and general public info. Also, add licensing information and a code of conduct file.

See [here](https://github.com/CharlVS/atomicdex-mobile/blob/revamp-project-readme/README.md) for a preview of the changes to the `README.md`.

### CODE REVIEW

**Comment by CharlVS** [comment]
Revamp of the technical build/setup guide will be a separate PR.

**Comment by CharlVS** [comment]
@smk762 @ca333 This is ready for review.

File: README.md [hunk 1]
```diff
@@ -11,22 +9,52 @@
 ![GitHub repo size](https://img.shields.io/github/repo-size/komodoplatform/atomicdex-mobile)
 ![Twitter Follow](https://img.shields.io/twitter/follow/komodoplatform?style=social)
 
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/10762374/231314925-93ec93d2-63f7-4361-8d25-71c0e1bb3e38.png" alt="adex swap screenshot" />
+Welcome to the AtomicDEX Mobile Wallet open-source repository! This cutting-edge project is brought to you by Komodo, providing a mighty non-custodial wallet and decentralised exchange all in one mobile app. Take control of your cryptocurrencies and trade seamlessly on your mobile phone or tablet with AtomicDEX!
+
+<p float="center">
```
Comment (smk762) [inline README.md:14]: Might be worth splitting this into two lines of 3 images to keep them centered.

File: README.md [hunk 2]
```diff
@@ -11,22 +9,52 @@
...
 # NB (Forkers/contributors):
-This repository is currently in the procesundergoing safety and Flutter version upgrade. Expect major merge conflicts in the near future for any files updated from this repository. Please note that this software is under active development and is provided "as is" without any warranties or guarantees. Use at your own risk, as the authors and maintainers shall not be held liable for any issues, damages, or losses resulting from its use.
+
+This repository is currently in the process of undergoing safety and Flutter version upgrades. Expect major merge conflicts in the near future for any files updated from this repository. Please note that this software is under active development and provided "as is" without warranties or guarantees. Use at your own risk, as the authors and maintainers shall not be held liable for any issues, damages, or losses resulting from its use.
 
 ## Getting Started
 
 Build requires up-to-date version of coins file from https://github.com/KomodoPlatform/coins
 
 Commit hash and sah256sum of coins file is specified in `coins_ci.json`.
```
Comment (smk762) [inline README.md:55]: ~sah256sum~
`sha256sum`

File: README.md [hunk 3]
```diff
@@ -11,22 +9,52 @@
...
-You may download one manually or use `fetch_coins.sh` script on linux and macOS,
- `fetch_coins.ps1` powershell script on Windows.
+You may download one manually or use `fetch_coins.sh` script on Linux and macOS,
```
Comment (smk762) [inline README.md:56]: ~You may download one manually or use `fetch_coins.sh` script on Linux and macOS, `fetch_coins.ps1` PowerShell script on Windows.~
You may download one manually or use the `fetch_coins.sh` script on Linux and macOS, or the `fetch_coins.ps1` PowerShell script on Windows.

File: README.md [hunk 4]
```diff
@@ -11,22 +9,52 @@
...
+ `fetch_coins.ps1` PowerShell script on Windows.
 
 `fetch_coins` script depends on sha256sum and jq utils:
```
Comment (smk762) [inline README.md:59]: ~`fetch_coins` script depends on sha256sum and jq utils:~
`The `fetch_coins` script depends on sha256sum and jq utils:`

File: README.md [hunk 5]
```diff
@@ -49,54 +77,37 @@ flutter run --dart-define=screenshot=true
...
-    git pull
-    git reset --hard
-    git checkout -f v1.14.3
+See [our wiki](https://github.com/KomodoPlatform/atomicdex-mobile/wiki/Project-Setup#android-builds-from-scratch) here for more thorough project setup steps. Besides installing the API binary, AtomicDEX Mobile is set up similarly to any other cloned Flutter project.
 
 ### Kotlin vs Flutter
 
-In Android Studio (3.6.2) the latest Kotlin plugin (1.3.71) doesn't work with Flutter “1.12.13+hotfix.7”. To fix it - [uninstall the latest Kotlin](https://github.com/flutter/flutter/issues/52077#issuecomment-600459786) - then the Kotlin version 1.3.61, bundled with the Android Studio, will reappear.
-
+In Android Studio (3.6.2), the latest Kotlin plugin (1.3.71) doesn't work with Flutter “1.12.13+hotfix.7”. To fix it - [uninstall the latest Kotlin](https://github.com/flutter/flutter/issues/52077#issuecomment-600459786) - then the Kotlin version 1.3.61, bundled with the Android Studio will reappear.
```
Comment (smk762) [inline README.md:102]: In Android Studio (3.6.2), the latest Kotlin plugin (1.3.71) doesn't work with Flutter `1.12.13+hotfix.7`.
To fix it
 - [uninstall the latest Kotlin](https://github.com/flutter/flutter/issues/52077#issuecomment-600459786) 
 - The Kotlin version 1.3.61, bundled with the Android Studio will reappear.

File: README.md [hunk 6]
```diff
@@ -117,13 +128,13 @@ Clone the latest version of [coins](https://github.com/KomodoPlatform/coins)
...
+Open the cloned repository and run the script below in the terminal in the repo folder.
 
 ```bash
 python3 utils/generate_app_configs.py
 ```
 
-Copy the generated `coins_config.json` file in Utils folder and paste inside assets/ folder in AtomicDEX-mobile project
+Copy the generated `coins_config.json` file from the Utils folder and paste it inside the `assets/` folder in the AtomicDEX Mobile project.
 
 ## Audio samples sources
```
Comment (smk762) [inline README.md:140]: Please update testplan link to https://docs.google.com/spreadsheets/d/1jeIkGe2CmJ7YmuoVi6Rlc9KRr3wiBPf44Qy0Nd8qtOY/edit?usp=sharing

**Comment by CharlVS** [comment]
@smk762 Thanks for catching the points I missed. It's ready for re-review.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+
+
+# Contributing Guide (Placeholder)
```
Comment (smk762) [inline CONTRIBUTING.md:3]: Is it still a placeholder?

**Review by ca333 — APPROVED** [review]

**Pull request merged**